packages/casl-angular/package.json

```diff
@@ -46,9 +46,9 @@
   "author": "Sergii Stotskyi <sergiy.stotskiy@gmail.com>",
   "license": "MIT",
   "peerDependencies": {
-    "@angular/core": "^12.0.0",
+    "@angular/core": "^12.0.0 || ^13.0.0-0",
     "@casl/ability": "^3.0.0 || ^4.0.0 || ^5.1.0",
-    "rxjs": "^6.5.3",
+    "rxjs": "^6.5.3 || ^7.0.0",
     "tslib": "^2.0.0"
   },
   "devDependencies": {
```

Comment:
in devDependencies you need to update `rxjs`, so then when test pass I will know it uses the correct version of rxjs